analytics/lib/counts.py

```diff
@@ -114,7 +135,12 @@ def do_aggregate_to_summary_table(stat, end_time, interval):
                'property' : stat.property,
                'interval' : interval}
 
+        start = time.time()
         cursor.execute(realmcount_query, {'end_time': end_time})
+        end = time.time()
+        rows_changed = cursor.rowcount
```

Comment:
I would just put cursor.rowcount into the logging.info args below, without defining a new variable.